homeassistant/components/supla/__init__.py

```diff
@@ -39,25 +49,26 @@
 )
 
 
-def setup(hass, base_config):
+async def async_setup(hass, base_config):
     """Set up the Supla component."""
 
     server_confs = base_config[DOMAIN][CONF_SERVERS]
 
-    hass.data[SUPLA_SERVERS] = {}
-    hass.data[SUPLA_CHANNELS] = {}
+    hass.data[DOMAIN] = {SUPLA_SERVERS: {}, SUPLA_COORDINATORS: {}}
 
     for server_conf in server_confs:
 
         server_address = server_conf[CONF_SERVER]
 
         server = SuplaAPI(server_address, server_conf[CONF_ACCESS_TOKEN])
+        server.update_interval = server_conf[CONF_SCAN_INTERVAL]
```

Comment:
what does api do with information since DataUpdateCoordinator is handling call interval?

Reply:
Yes, it's unnecessary now, removed.